pkg/rancher-desktop/backend/lima.ts

```diff
@@ -752,13 +752,18 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     const VMNETDir = path.join(VMNET_DIR, 'bin');
     const pathList = (process.env.PATH || '').split(path.delimiter);
     const newPath = [binDir, VMNETDir].concat(...pathList).filter(x => x);
+    const env = process.env;
```

Comment:
This is a reference, not a copy; did you mean to make a copy here?
```suggestion
    const env = { ...process.env };
```
```suggestion
    const env = structuredClone(process.env);
```

Reply:
Yes, good catch!